code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -333,8 +333,12 @@ var/bomb_set
 	yes_code = 0
 	safety = 1
 	update_icon()
-	for(var/mob/M in player_list)
-		M << 'sound/machines/Alarm.ogg'
+	if(prob(1))
+		for(var/mob/M in player_list)
```

Comment:
I think it might be easier if you do something like 

```
var/nuke_sound = 'sound/machines/Alarm.ogg'

if (prob(1))
    nuke_sound = 'sound/misc/imnuclear.ogg'

for(var/mob/M in player_list)
    M << nuke_sound
```